code/modules/unit_tests/README.md

```diff
@@ -39,7 +39,12 @@ Open `code/_compile_options.dm` and uncomment the following line.
 //#define UNIT_TESTS			//If this is uncommented, we do a single run though of the game setup and tear down process with unit tests in between
 ```
 
-Then, run tgstation.dmb in Dream Daemon. Don't bother trying to connect, you won't need to. You'll be able to see the outputs of all the tests. You'll get to see which tests failed and for what reason. If they all pass, you're set!
+There are 2 ways to run unit tests
```

Comment:
There's more than two ways I suppose (I just use the text explorer extension)

Reply:
I forgot about the test explorer good point. I'll update it